results/google__text-embedding-005/external/ArguAna.json

```diff
@@ -0,0 +1,38 @@
+{
+  "dataset_revision": null,
```

Comment:
Seems to be missing for multiple datasets.

Reply:
Added!